### DISCUSSION TITLE
How to stop discord bot being triggered by keyword for specific scenarios

### DISCUSSION THREAD

**Discussion by rock3tship** [community/community#21457] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I don’t have much experience on these sites or in coding. This is in python.</p>
<p>my question is, if i have my discord bot coded to send a reply to ‘simon’ anywhere in a sentence and also different replies for ‘hello simon’ , can i stop the ‘simon’ responses sending?</p>
<p>the code looks like this so far (I put -but not- to show what i was trying to get at)<br>
if (‘simon’) in message.content.lower() -BUT NOT- (‘hello simon’) in message.content.lower():<br>
await message.channel.send(random.choice(simon_msgs))</p>
<p>(i have the simon_msgs sorted i just need it to not send simon messages when hello is infront of it)</p>
<p>thank you</p>

**Comment by mpboom** [comment, marked as answer]
<p>You could try using an <code>elif</code> statement:</p>
<pre><code class="lang-python">if 'hello simon' in message.content.lower():
    pass  # Handle "Hello, Simon" case
elif 'simon' in message.content.lower():
    pass  # Handle "Simon", but not "Hello, Simon" case
else:
   pass  # No "Simon" at all
</code></pre>
<p>If you’d purely like to filter out that one condition, you could do it like this:</p>
<pre><code class="lang-python">if 'simon' in message.content.lower() and not 'hello simon' in message.content.lower()
</code></pre>
<p>(see I just chained the statement using <code>and not</code>?)</p>

**Comment by rock3tship** [comment]
<p>Thank you so much! works perfectly :slight_smile:</p>
